pages/3_Compare_Cases.py

```diff
@@ -44,14 +26,14 @@
             eid, row_ids, row_id_text="row_id_comp", prefix="second", default=1
         )
         entity_difference.view_instructions(use_row_ids=True)
-elif tab == "2":
+else:
     filtering.view_entity_select(eid_text="eid", prefix="first", default=0)
     filtering.view_entity_select(eid_text="eid_comp", prefix="second", default=1)
     entity_difference.view_instructions()
 
 # Global options ------------------------------
 filtering.view_filtering()
-if tab == "1":
+if config.USE_ROWS:
     if len(row_ids) >= 2:
         if st.session_state["row_id"] == st.session_state["row_id_comp"]:
             st.warning("Please select two different prediction times!")
```

Comment:
One thing I missed earlier - sibylapp is built to be completely application agnostic, so the rows may not be timestamps. Though this will require an update to sibylapi to send the label for rows, so for this PR we can leave it as is